fvm/evm/testutils/contract.go

```diff
@@ -25,49 +25,74 @@ type TestContract struct {
 	require.NoError(tb, err)
 	return &TestContract{
 		Code: `
-		pragma solidity >=0.7.0 <0.9.0;
+		// SPDX-License-Identifier: GPL-3.0
```

Comment:
should we maybe start extracting solidity code and bytecode into separate files, like `Storage.sol` which are then embeded into the Go code using the `embed`. I prefer it because you have syntax highlighting possible, you can use the file directly to compile to bytecode, I even think we should in the future generate the bytecode file using a make action.